lib/identifier/uuid/controls/incrementing.rb

```diff
@@ -16,7 +16,7 @@ def self.example(i=nil, sample: nil)
             third_prefix = third_prefixes[0]
           end
 
-          "#{first_octet}-4000-#{third_prefix}000-0000-000000000000"
+          "#{first_octet}-0000-4000-#{third_prefix}000-000000000000"
```

Comment:
That means that the "third prefix" is not the third but the fourth. Should change the variable name.